src/contracts/libraries/SlashingLib.sol

```diff
@@ -1,11 +1,15 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.27;
 
+import "@openzeppelin/contracts/utils/math/Math.sol";
```

Comment:
Nice! pls check out AllocationManager and DelegationManager for more potential overflows

Reply:
AM done, DM doesn't need it.